src/login/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { updateIntl } from "../../../reducers/Internationalisation";
+import { AVAILABLE_LANGUAGES, LOCALIZED_MESSAGES } from "globals";
+import { useDashboardAppDispatch, useDashboardAppSelector } from "dashboard-hooks";
+import { FormattedMessage } from "react-intl";
+
+const Footer = (): JSX.Element => {
+  const browserLocale = useDashboardAppSelector((state) => state.intl.locale);
+  const eduidHomeUrl = useDashboardAppSelector((state) => state.config.eduid_site_url);
+  const dispatch = useDashboardAppDispatch();
+  const faqUrl = browserLocale === "en" ? `/en/faq.html` : `/faq.html`;
+  const messages = LOCALIZED_MESSAGES as unknown as { [key: string]: { [key: string]: string } };
+
+  let translateTo: string[][] = [];
+  let locale = "";
+  let language = "";
+
+  if (AVAILABLE_LANGUAGES !== undefined) {
+    translateTo = AVAILABLE_LANGUAGES.filter((lang) => lang[0] !== browserLocale);
+    locale = translateTo[0][0];
+    language = translateTo[0][1];
+  }
+
+  // TODO: Figure out what this code is used for, from Footer container
```

Comment:
I think you can safely remove these lines. It is from when we were deploying the first React-based frontend.